src/pages/games.html

```diff
@@ -0,0 +1,217 @@
+                    <p class="About-review-container-paragraph">At first, Hulk was going to be gray but due to printing
+                        color problems it was changed to green.
+                        The precision of the printing presses was not so great at that time</p>
+                </div>
+            </div>
+            <div class="About-review-container-row">
+                <div class="About-border-right About-review-container-box">
+                    <h3 class="About-review-container-text">Mario Puzo, I try to write comics</h3>
+                    <p class="About-review-container-paragraph">The Godfather writer tried several times to write comics
+                        for Marvel, but was rejected due to lack
+                        of creativity. Who knows if he had continued what types of comics he would have released</p>
+                </div>
+                <div class="About-review-container-box">
+                    <h3 class="About-review-container-text">Weak points</h3>
+                    <p class="About-review-container-paragraph">Another curiosity that many have not stopped at is that
+                        most superheroes, all in the case of
+                        Marvel, have a weak point that exposes them and leaves them vulnerable to enemies.</p>
+                </div>
+            </div>
+            <div class="About-review-container-row About-border-up">
+                <div class="About-border-right About-review-container-box">
+                    <h3 class="About-review-container-text">Comic and Hitler</h3>
+                    <p class="About-review-container-paragraph">Some of the most famous comics in history were part of
+                        North American propaganda against the
+                        advance of Nazism, and good proof of this is that, in his first appearance, Captain America is
+                        hitting none other than Adolf in the face. Hitler.</p>
+                </div>
+                <div class="About-review-container-box">
+                    <h3 class="About-review-container-text">Superhero skills</h3>
+                    <p class="About-review-container-paragraph">His father and creator of other classics such as
+                        Spiderman, Ironman, Hulk or the X-Men, the great
+                        Stan Lee, hosted a reality show in the mid-2000s, whose objective was none other than to find
+                        people with talents or abilities typical of The Superheroes.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="About-subscribe">
+        <div class="About-subscribe-box">
+
+            <div class="About-subscribe-title">
+                <h2 class="About-subscribe-title-text">WRITE US AT SMBS SOLUTIONS</h2>
+                <p class="About-subscribe-title-paragraph">We are your best friend designing the algorithm of your
```

Comment:
The name of this class could be **About-subscribe-paragraph**

Reply:
done